src/libra_py/tsh.py

```diff
@@ -915,7 +915,7 @@ def hopping(Coeff, Hvib, istate, sh_method, do_collapse, ksi, ksi2, dt, T):
     if sh_method==0:
         g = compute_hopping_probabilities_mssh(Coeff)
     elif sh_method==1:
-        g = compute_hopping_probabilities_fssh(Coeff, Hvib, dt)
+        g = compute_hopping_probabilities_fssh(Coeff, Hvib, dt, do_rescale, T)
```

Comment:
do_rescale is to rescale velocities. There are no velocities in the NBRA-type methods, so there is no need to do this. In addition, this change breaks the backward compatibility